webapp/src/api.rs

```diff
@@ -161,11 +164,23 @@ pub struct Payment {
     fee: u64,
 }
 
-pub async fn send_payment(params: Json<Payment>) -> Result<(), AppError> {
+pub async fn send_payment(
+    State(state): State<Arc<AppState>>,
+    Json(params): Json<Payment>,
+) -> Result<(), AppError> {
+    if !state
```

Comment:
With this we force users to define whitelisted addresses. Should we allow for having an empty list of whitelisted addresses to bypass this check?

Reply:
I will make it more explicit and add a cli flag to enable or disable that feature.